.github/workflows/python_wheels.yml

```diff
@@ -0,0 +1,330 @@
+name: Python
```

Comment:
do we need a small description at the beginning of this file or do you think if someone just reads it is clear enough?